xchange-stream-core/src/test/java/info/bitrich/xchangestream/util/BookSanityCheckerTest.java

```diff
@@ -0,0 +1,123 @@
+package info.bitrich.xchangestream.util;
+
+import info.bitrich.xchangestream.util.BookSanityChecker;
+import org.junit.Assert;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+import org.knowm.xchange.currency.CurrencyPair;
+import org.knowm.xchange.dto.Order;
+import org.knowm.xchange.dto.trade.LimitOrder;
+import org.knowm.xchange.dto.marketdata.OrderBook;
+
+import java.math.BigDecimal;
+import java.util.ArrayList;
+import java.util.Date;
+
+import static java.lang.String.format;
+
+public class BookSanityCheckerTest {
+
+  @Rule
+  public final ExpectedException thrown = ExpectedException.none();
```

Comment:
I might be missing something here but I can't see that you are throwing an exceptions

Reply:
Fixed.